docs/install/README.md

```diff
@@ -76,6 +87,8 @@ $ docker pull diygod/rsshub
 
 ## Docker 部署
 
+> 如需启用 puppeteer，请改用 `diygod/rsshub:chromium-bundled` (仅限 x86\_64 架构)
+
 ### 安装
 
 运行下面的命令下载 RSSHub 镜像
```

Comment:
````suggestion
### 安装

运行下面的命令下载 RSSHub 镜像


```bash
$ docker pull diygod/rsshub
```

运行下面的命令下载内置 puppeteer 的 RSSHub 镜像

...
````

Reply:
EACH command needs to replace `diygod/rsshub` with `diygod/rsshub:chromium-bundled`, not only in the pull stage.
I consider these prompts are enough, no need to prompt more:

https://github.com/DIYgod/RSSHub/blob/8a9b6e98a16427558280a4a5f40777b21ed08c4a/docs/install/README.md?plain=1#L36

https://github.com/DIYgod/RSSHub/blob/8a9b6e98a16427558280a4a5f40777b21ed08c4a/docs/install/README.md?plain=1#L90